src/features/chains/utils.ts

```diff
@@ -36,21 +37,36 @@ export function getChainByRpcUrl(multiProvider: MultiProtocolProvider, url?: str
  * Returns an object that contains the amount of
  * routes from a single chain to every other chain
  */
-export function getAllTokenRoutesFromChain(
+export function getNumRoutesWithSelectedChain(
   warpCore: WarpCore,
-  origin: ChainName,
```

Comment:
@Xaroz your first version was close but technically num routes _from_ a chain doesn't _necessarily_ equal the num routes _to_ that chain, so we need to treat it as an origin/destination depending on which field it's coming from